content/1_docs/2_cookbook/7_extensions/0_first-panel-view/recipe.txt

```diff
@@ -0,0 +1,358 @@
+    }
+};
+</script>
+```
+
+If we now return to the Panel, our view has changed and will show the cards component with a single empty card in it.
+
+(screenshot: panel-view-with-card.png)
+
+
+## Connecting the data
+
+We are almost there. Here is the final `Review.vue` component which we will explain in more detail in a minute.
+
+```js "/site/plugins/moviereviews/src/components/Review.vue"
+<template>
+  <div class="k-reviews">
+    <k-cards data-size="medium">
+      <k-card
+        v-for="(review) in reviews"
+        :key= review.display_title
+        v-bind="review"
+        :info= review.byline
+        :text= review.summary_short
+        :image="{
+          ratio: '3/2',
+          cover: true,
+          back: 'pattern',
+          url: review.multimedia.src
+        }"
+        v-on="$listeners"
+        :link= review.link.url
+        target="_blank"
+      />
+    </k-cards>
+ 
+  </div>
+</template>
+
+<script>
+
+export default {
+  data() {
+    return {
+      reviews: [],
+    };
+  },
+   created() {
+    this.load();
+  },
+  methods: {
+    load() {
+      this.$api
+        .get("moviereviews")
+        .then(reviews => {
+          this.reviews = reviews;
+        });
+    }
+  }
+}
+</script>
+```
+
+Let's first look at the script part:
+
+```js
+<script>
+export default {
+  data() {
+    return {
+      reviews: [],
+    };
+  },
+  created() {
+    this.load();
+  },
+  methods: {
+    load() {
+      this.$api
+        .get("moviereviews")
+        .then(reviews => {
+          this.reviews = reviews;
+        });
+    }
+  }
+}
+</script>
+```
+
+
+Within the `export` object, we have to define the data our component returns as a function.
+
+```js
+data() {
+return {
+    reviews: [],
+};
+},
+```
+
+Once the Vue instance is created, we call the `load()` method.
+
+```js
+created() {
+this.load();
+},
+```
+
+The `load()` method calls the `moviereviews` route from the API, and then assigns the result to `this.reviews`.
+
+```js
+methods: {
+load() {
+    this.$api
+    .get("moviereviews")
+    .then(reviews => {
+        this.reviews = reviews;
+    });
+}
+}
+```
+
+We can now loop through the data an cread a card for each review:
+
+```js
+<k-cards data-size="medium">
+    <k-card
+    v-for="review in reviews"
+    :key= review.display_title
+    v-bind="review"
+    :info= review.byline
+    :text= review.summary_short
+    :image="{
+        ratio: '3/2',
+        cover: true,
+        url: review.multimedia.src
+    }"
+    v-on="$listeners"
+    :link= review.link.url
+    target="_blank"
+    />
+</k-cards>
+```
+
+Here we fill the props of the (link: docs/reference/plugins/ui/card text: card component) with life. We loop through each review `( v-for="review in reviews")`and assign the data items to the `info`, `text` and `image` props. We also add a `link` to the original review and add the `$listeners` instance property, so that we are directed to the review source when we click on a card.
```

Comment:
A space is missing after `( v-for="review in reviews")`. Also i think the parens should be outside of the code backticks.

Reply:
✅